tests/test_ssh_scan.py

```diff
@@ -0,0 +1,13 @@
+from lib.target import Target
+from lib.ssh_scan import SSHScanTask
+import json
+
+
+class TestSSHScanTask(object):
+    def test_SSHScan(self):
+        target = Target("ssh.mozilla.com")
+        task = SSHScanTask(target)
+        result = task.run()
+        stdout, _ = result.communicate()
```

Comment:
This whole test is pretty hacky at the moment, and it will fail; as in SSHScanTask, `ssh_scan` dumps results in a file (with `-o`), therefore it does not return anything to `stdout`.

A better way to do this is perhaps using subprocess.call, without shell execution (see https://github.com/caggle/vautomator-standalone/issues/5) and specifying stdout as a file, instead of using the tools' output flags. So that we can operate/assert on a tool output in its relevant test.